packages/manager/modules/sms/src/sms/batches/history/delete/delete.controller.js

```diff
@@ -0,0 +1,25 @@
+export default class TelecomSmsBatchesHistoryDeleteCtrl {
+  /* @ngInject */
+  constructor($http, $translate, SmsService, TucToast) {
+    this.isLoading = false;
+    this.$http = $http;
+    this.$translate = $translate;
+    this.SmsService = SmsService;
+    this.TucToast = TucToast;
+  }
```

Comment:
**nitpick:** `TucToast` is not used in the controller. So it can be removed.
```suggestion
  constructor($http, $translate, SmsService) {
    this.isLoading = false;
    this.$http = $http;
    this.$translate = $translate;
    this.SmsService = SmsService;
  }
```